checkstyle/checkstyle.xml

```diff
@@ -9,7 +9,7 @@
 
   <!-- header -->
   <module name="RegexpHeader">
-    <property name="header" value="/\*\*\n * Copyright \(c\) Dell Inc\.\, or its subsidiaries\. All Rights Reserved\."/>
+    <property name="header" value="/\*\*\n * Copyright \(c\) Pravega Authors. All Rights Reserved\."/>
```

Comment:
Updating the rule with this regex requires we proactively update all file headers throughout the repo. Do we do this sweep change in this PR? Or do we allow the regex to accept both headers?